README.md

```diff
@@ -1,28 +1,108 @@
 
-1. `git clone https://github.com/MeteoSwiss-APN/pytrajplot.git`
-2. `make install`
-3. `conda activate pytrajplot`
-4. `./tests/test_pytrajplot.sh`
+Having Conda installed is a prerequisite for the further installation
+process. If that is not the case, install the latest Miniconda version with
+
+    tools/setup_miniconda.sh
+
+or install it manually from the
+[miniconda webpage](https://docs.conda.io/en/latest/miniconda.html). Afterwards, follow the
+instructions here below to clone the GitHub repository, set up a conda environment and test all
+possible use-cases.
+
+Create an environment and install the package dependencies with the provided script `setup_env.sh`.
+Check available options with
+```bash
+tools/setup_env.sh -h
+```
+We distinguish pinned installations based on exported (reproducible) environments,
+and free installations, where the installation
+is based on top-level dependencies listed in `requirements/requirements.txt`.
+A pinned installation is done with
+
+    tools/setup_env.sh -n <package_env_name>
+
+If you start developing, you might want to do an unpinned installation and export the environment:
+
+    tools/setup_env.sh -u -e -n <package_env_name>
+
+*Hint*: Use the flag `-m` to speed up the installation using mamba. You will of course have to
+install mamba first. We recommend to install mamba into your base
+environment `conda install -c conda-forge mamba`. If you install mamba in another (maybe dedicated)
+environment, environments installed with mamba will be located
+in `<miniconda_root_dir>/envs/mamba/envs`, which is not very practical.
+
+The package itself is installed with `pip`.
+
+    conda activate <package_env_name>
+    pip install .
+
+For development, install the package in editable mode:
+
+    conda activate <package_env_name>
+    pip install --editable .
+
+*Warning:* Make sure you use the right pip, i.e. the one from the installed conda environment (`which pip` should point to something like `path/to/miniconda/envs/<package_env_name>/bin/pip`).
+
+Once your package is installed, run the tests by typing:
+
+```
+conda activate <package_env_name>
+pytest
+```
+
+If the tests pass, you are good to go.
+If developing, make sure to update the requirements file
+and export your environment after installation
+every time you add new imports while developing.
+
+### Shortcut using Makefile
+
+Instead of intalling the environment and the package with the above commands,
+you may alternatively use the available Makfile.
+Note however that providing a Makefile is no longer supported by the MeteoSwiss blueprint and the Makefile may therefore be removed in a future release.
```

Comment:
Fixed it like that:
"Note that the MeteoSwiss blueprint no longer supports providing a Makefile. As a result, the Makefile will be deprecated in future releases."

Reply:
This means it is not yet deprecated, thus users may use it. Is that correct?

Reply:
Maybe you're trying to say this? "Using the Makefile is deprecated, because MeteoSwiss' blueprint stopped support. It will be removed in a future version. Please migrate to ...."
I think if something is deprecated it should be mentioned how users can migrate.